storefront/test/integration/workarea/storefront/checkouts_integration_test.rb

```diff
@@ -345,6 +345,12 @@ def test_no_valid_shipping_options
 
         assert(flash[:error].present?)
       end
+
+      def test_not_setting_a_blank_tracking_email
```

Comment:
Is this test right? It sets a cookie and tests that it is still set?

Reply:
Yes, without the change in the checkouts controller, it will blank out the tracking email incorrectly.

Reply:
The test is to assert the email doesn't get cleared.